src/stactools/jrc_gsw/assets.py

```diff
@@ -1,116 +1,145 @@
+                    "and (2000-2020)"
+                ),
+                "type": pystac.MediaType.COG,
+                "roles": ["data"],
+            }
+        ),
+        SEASONALITY_KEY: AssetDefinition(
+            {
+                "title": "Seasonality",
+                "description": (
+                    "Number of months that water was present from January 2020 to "
+                    "December 2020"
+                ),
+                "type": pystac.MediaType.COG,
+                "roles": ["data"],
+                "start_datetime": SEASONALITY_START_TIME,
+                "end_datetime": SEASONALITY_END_TIME,
+            }
+        ),
+        RECURRENCE_KEY: AssetDefinition(
+            {
+                "title": "Recurrence",
+                "description": ("Frequency with which water returns from year to year"),
+                "type": pystac.MediaType.COG,
+                "roles": ["data"],
+            }
+        ),
+        TRANSITIONS_KEY: AssetDefinition(
+            {
+                "title": "Transitions",
+                "description": (
+                    "Categorical change in surface water status from 1984 to 2020"
+                ),
+                "type": pystac.MediaType.COG,
+                "roles": ["data"],
+            }
+        ),
+        EXTENT_KEY: AssetDefinition(
+            {
+                "title": "Maximum Water Extent",
+                "description": (
+                    "Binary indicator of whether water was ever present (from 1984-2020)"  # noqa
+                ),
+                "type": pystac.MediaType.COG,
+                "roles": ["data"],
+            }
+        ),
+    },
+    MONTHLY_HISTORY["ID"]: {
+        MONTHLY_HISTORY_KEY: AssetDefinition(
+            {
+                "title": "Monthly History",
+                "description": ("Historical water detection on a month-by-month basis"),
+                "type": pystac.MediaType.COG,
+                "roles": ["data"],
+            }
+        )
+    },
+    YEARLY_CLASSIFICATION["ID"]: {
+        YEARLY_CLASSIFICATION_KEY: AssetDefinition(
+            {
+                "title": "Yearly Classification",
+                "description": (
+                    "Year-by-year classification of the seasonality of water based on the occurrence values detected throughout the year."  # noqa
+                ),
+                "type": pystac.MediaType.COG,
+                "roles": ["data"],
+            }
+        ),
+    },
+}
+
+MONTHLY_RECURRENCE_ASSETS = {}
+for i in range(1, 13):
+    month_name = datetime.strptime(str(i), "%m").strftime("%B")
+    MONTHLY_RECURRENCE_ASSETS[f"{MONTHLY_RECURRENCE_KEY}{i}"] = AssetDefinition(
         {
-            "title": "Monthly Recurrence",
+            "title": f"Monthly Recurrence ({month_name})",
             "description": (
                 "Monthly measures of the seasonality of water based on the occurrence values detected in that month over all years."  # noqa
```

Comment:
This is outdated. Because we've split these items by month, I believe we should have the same title and description for all of these assets (i.e. I don't think we should include month in either the title or description).